.github/workflows/dependabot-changeset-maker.yml

```diff
@@ -0,0 +1,72 @@
+name: "Dependabot changeset maker"
+on:
+- pull_request_target
+
+jobs:
+  generate-changeset:
+    runs-on: ubuntu-latest
+    if: github.actor == 'dependabot[bot]'
```

Comment:
Still feeling we should have the fork check, just to not confuse people in case they run dependabot on their forks

Reply:
Added 👍